### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ New Dynamic Web App (Dark Theme)_Color Contrast: In Dark theme, Color contrast ratio for the text message present under the ‘Configuration’ section is less than 4.5:1

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3332]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.4.3; #Color Contrast; #C+E-H; #Java tooling for IntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS1.4.3

**Note: Similar issue is found for 'Create New Web App' popup for 'Create App Service' text**

**Repro Steps:** 
1) Open Java EE - Eclipse application and change the theme to dark (WindowsPreferencesGeneralappearancesSelect Dark theme) 
2) Install 'Azure Toolkit for Eclipse' from Marketplace settings/preferences popup
3) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
4) Tab till File-->New-->Project--->Dynamic Web Project
5) Open Color Contrast Analyzer
6) Verify the Color contrast ratio for the text message present under the ‘Configuration’ section

**Actual Result:** 
In Dark theme, Color contrast ratio for the text message present under the ‘Configuration’ section is 3.9:1 which is less than 4.5:1

**Expected Result:** 
In Dark theme, Color contrast ratio for the text message present under the ‘Configuration’ section should be greater than or equal to 4.5:1

**User Impact:** 
Users with low vision will not be able to see the text properly if the color contrast ratio is less than required

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

![Bug_11_Color contrast issue](https://user-images.githubusercontent.com/41109135/62204984-fef8ab80-b3ab-11e9-9dab-f29b9f3917c0.png)

![Create new Web App](https://user-images.githubusercontent.com/41109135/62210640-40905300-b3ba-11e9-8c3a-e99079f37e0c.png)

**Comment by Flanker32** [comment]
The fonts of label and title are controlled by Eclipse theme which is out of the scope of IDE Toolkits, and eclipse provide multiple theme and this issue will not happen in other theme, for example, Light, here are the screen shots.

*Dynamic Web Project*
![image](https://user-images.githubusercontent.com/12445236/64249541-79d25a80-cf46-11e9-9b78-e1e52c1462de.png)

*Create App Service*

![image](https://user-images.githubusercontent.com/12445236/64250078-ce2a0a00-cf47-11e9-870e-447e9b3e5be5.png)

**Comment by Flanker32** [comment]
- Importance of experience: Medium,  Users with low vision will not be able to see the message properly.
- Quality of experience: not impacted. Eclipse provide alter theme for users, for instance, light, which has high contrast ratio.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
Tracking in [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076) here, @Tanushagottipati please have a review.

**Comment by Tanushagottipati** [comment]
External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=550838

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by Manisha370** [comment]
Similar issue is observed in "New HDInsight Scala Project" page
In dark theme, the color contrast ratio for the text is less than 4.5:1

![New HDInsight Scala Project_In dark theme the color contrast ratio is very poor](https://user-images.githubusercontent.com/51435937/79110747-b85fab80-7d98-11ea-92ff-438a45cf91e4.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749662
